cmake/CMakeLists.txt

```diff
@@ -401,6 +400,14 @@ foreach(SIMPLE_LIB REAX MEAM POEMS USER-ATC USER-AWPMD USER-COLVARS USER-H5MD
       target_include_directories(awpmd PUBLIC ${LAMMPS_LIB_SOURCE_DIR}/awpmd/systems/interact ${LAMMPS_LIB_SOURCE_DIR}/awpmd/ivutils/include)
     elseif(PKG_LIB STREQUAL h5md)
       target_include_directories(h5md PUBLIC ${LAMMPS_LIB_SOURCE_DIR}/h5md/include)
+    elseif(PKG_LIB STREQUAL colvars)
+      file(GLOB_RECURSE LEPTON_SOURCES ${LAMMPS_LIB_SOURCE_DIR}/colvars/lepton/src/*.cpp)
```

Comment:
Why that double include the cpp fils in `colvars/lepton/src`as there is already a `file(GLOB_RECURSE ${PKG_LIB}_SOURCES ... ${LAMMPS_LIB_SOURCE_DIR}/${PKG_LIB}/*.cpp)`?

And do the Makesfiles build this a two libraries as well?

Maybe `USER-COLVARS` is better build outside of the `SIMPLE_LIB` loop.

Reply:
Simplified it.